### PR TITLE
Remove some unnecessary dependencies

### PR DESCRIPTION
**Pull request by hlinnaka**
These are not actually used anywhere. The build was failing on my system on an 'openblas' link error, and this conveniently fixes that :-).

### CODE REVIEW

**Review by sgichohi — APPROVED** [review]

**Comment by cevian** [comment]
Ahh sorry, forgot about these when removing PQ.

**Review by cevian — APPROVED** [review]

**Comment by cevian** [comment]
@hlinnaka if you can sign the CLA I'll be happy to merge

**Comment by alejandrodnm** [comment]
H @hlinnaka, this PR is blocked by the CLA. It's a company requirement, as soon as you sign it we can merge it :) 

Thanks.

**Comment by cevian** [comment]
This was done in another PR

**Pull request closed without merging**
